### ISSUE TITLE
support customize behavior such as rename @timestamp disable <value-passed-to-console-log-json-was-null>

### ISSUE THREAD

**Issue by dacapoday** [hiro5id/console-log-json#32]
Hi,

Is somebody know how to rename built-in log entry such as @timestamp, or disable some helper behavior such as replace null with hardcode placeholder <value-passed-to-console-log-json-was-null>?

**Comment by hiro5id** [comment]
I could add those capabilities.  I am away from office right now but this sounds like it could be useful configuration capabilities that could be added in relatively straight forward.

**Comment by hiro5id** [comment]
Hi it's been a while since I have had time to dedicate to this.  Could you give some more details of examples of how you would want to customize behavior ?

**Comment by hiro5id** [comment]
@dacapoday  Current status in console-log-json 6.2.0:

- Yes, LoggerAdaptToConsole(options) accepts configuration.
- Renaming built-in fields like @timestamp: supported via transformOutput.
- Customizing/removing helper placeholders like "<value-passed-to-console-log-json-was-null>": there is no dedicated flag for that, but you can rewrite or remove that message in transformOutput.
- Customizing the default level mapping of console.log / console.info / console.error: not directly configurable as a global mapping. The built-in mapping is still tied to the console method. What is
    supported:
  - logLevel to control the minimum output threshold
  - per-call override with { level: '...' }, for example console.log({ level: 'warn' }, 'message')
  - transformOutput can rewrite the final level field, but that does not change the internal filtering semantics

  Example:
```
  import { LoggerAdaptToConsole, LOG_LEVEL } from 'console-log-json';

  LoggerAdaptToConsole({
    logLevel: LOG_LEVEL.info,
    transformOutput: (obj) => {
      // rename @timestamp
      obj.timestamp = obj['@timestamp'];
      delete obj['@timestamp'];

      // remove or rewrite the null placeholder
      if (obj.message === '<value-passed-to-console-log-json-was-null>') {
        delete obj.message;
        // or: obj.message = null;
        // or: obj.message = 'null';
      }

      // optional: rewrite the emitted level label
      if (obj.level === 'info') {
        obj.level = 'notice';
      }

      return obj;
    }
  });
```
  So in short:

  - Yes for passing config to LoggerAdaptToConsole()
  - Yes for renaming fields
  - Yes, indirectly for changing/removing placeholder messages
  - No, not directly for globally remapping console methods to different default levels
